class-bu-slideshow.php

```diff
@@ -99,20 +97,48 @@ public function set_slides($slides = array()) {
 	}
 	
 	/**
+	 * Create new slideshow post
+	 * 
+	 * @return int
+	 */
+	protected function create_post(){
+		// Create post object
+		$post = array(
+		  'post_title'    => $this->name,
+		  'post_content'  => '',
+		  'post_status'   => 'publish',
+		  'post_type'     => 'bu_slideshow',
+		);
+
+		$result = $postID = wp_insert_post( $post );
+
+		if( is_wp_error( $result ) ) {
+		    error_log( sprintf( '[%s] Error creating post: %s Object: %s',
+				__FUNCTION__, $result->get_error_message(), var_export( $this, TRUE ) ) );
+			return FALSE;
+		} else {
+			if( false === add_post_meta( $postID, '_bu_slideshow', $this, true ) ){
```

Comment:
Perhaps a better name for the meta key here is `_bu_slideshow_slides`? Better represents the actual contents.